src/components/MapDetail/components/AssociatedSubjects/AssociatedSubjects.js

```diff
@@ -17,16 +33,53 @@ export default function AssociatedSubjects() {
       width='medium'
     >
       <Box align='center' direction='row' justify='between'>
-        <Text color='kelp'>Associated Zooniverse Subjects (32)</Text>
-        <Button
-          gap='0.2em'
-          icon={<FormNext size='small' />}
-          label={<Uppercase color='kelp' size='small'>See More</Uppercase>}
-          plain
-          reverse
-        />
+        <Heading color='kelp' level='6'>Associated Zooniverse Subjects ({subjects.length})</Heading>
+        {subjects.length > 10 && (
+          <Button
+            gap='0.2em'
+            icon={<FormNext size='small' />}
+            label={<Uppercase color='kelp' size='small'>See More</Uppercase>}
+            onClick={() => setShowSubjectsModal(true)}
+            plain
+            reverse
+          />
+        )}
+      </Box>
+      <Box
+        direction='row'
+        wrap
+      >
+        {firstTenSubjects.map((subject, i) => {
+          return (
+            <StyledButton
+              key={`ASSOCIATED_SUBJECT_${subject.id}`}
+              a11yTitle={`Subject ${subject.id}`}
+              margin={{ bottom: '0.5rem' }}
+              onClick={() => setActiveSubject(subject)}
+              plain
+            >
+              <Box
+                height='2.75em'
+                width='2.75em'
+              >
+                <Image
+                  alt={`Associated Subject ${i}`}
```

Comment:
```suggestion
                  alt={`Associated Subject ${subject.id}`}
```
The array index won't have meaning for someone using a screen reader. Let's also use the id here  to match the button label.

Reply:
Nice. Taking care of this in a separate PR I'm about to put up.